bench/bench_test.go

```diff
@@ -159,29 +160,29 @@ func benchmarkGetMetadatas(b *testing.B, n, hvkLen, mdLen int) {
 	b.StopTimer()
 }
 
-func getMultihashes(b *testing.B, mhs []multihash.Multihash, store *dhstore.PebbleDHStore) {
+func getMultihashes(b *testing.B, mhs []multihash.Multihash, store dhstore.DHStore) {
```

Comment:
I wanted to run the `bench_test` with the FDB store, however it kept failing with missing C bindings
```
/Users/ivan/go/pkg/mod/github.com/apple/foundationdb/bindings/go@v0.0.0-20230508183941-1098aa0251f4/src/fdb/database.go:26:11: fatal error: 'foundationdb/fdb_c.h' file not found
 #include <foundationdb/fdb_c.h>
          ^~~~~~~~~~~~~~~~~~~~~~
```

Am I missing something in my local environment / is there a way to run it on Mac?

Reply:
Yup, your setup is missing the c binding installation. Take a look at the changes I have made in the CI on how to install them. There are good docs on FDB website on installing binaries. 

You can also look at `Dockerfile` on how they are installed. Note that they are platform dependant. You will need arm64 for M1 if that's what you are running?

Reply:
Ok - will have a look thanks